### PR TITLE
[FIX] mail: allow overflow on the chatter

### PR DESCRIPTION
**Pull request by phenix-factory**
task-2333128

### CODE REVIEW

File: addons/mail/static/src/components/chatter/chatter.scss [hunk 1]
```diff
@@ -9,6 +9,10 @@
     overflow: auto;
 }
 
+.o_form_sheet .o_Chatter {
+  overflow: visible;
+}
```
Comment (alexkuhn) [inline addons/mail/static/src/components/chatter/chatter.scss:14]: Why we need `overflow: auto;` on `.o_Chatter` again? @seb-odoo

Comment (seb-odoo) [reply addons/mail/static/src/components/chatter/chatter.scss:14]: I've been asked the same question from AAB 2 months ago in the context of another problem.

If we can't' see anything wrong without it we should remover it.

Comment (seb-odoo) [reply addons/mail/static/src/components/chatter/chatter.scss:14]: So to answer the question: I don't know.

Comment (alexkuhn) [reply addons/mail/static/src/components/chatter/chatter.scss:14]: Ok, so I would remove `overflow: auto;` from `.o_Chatter` and check nothing's broken.
Should check community/enterprise and some special cases of chatter usage, like in Documents app.

**Review by alexkuhn — CHANGES_REQUESTED** [review]

**Comment by phenix-factory** [comment]
I have remove the overflow and tested:

- Enterprise/Community chatter (with toolbar for enterprise) 
- - large screen/little screen
- Enterprise/Community project "follow" bar
- Chat window Enterprise/Community

I didn't find any bug.

**Comment by seb-odoo** [comment]
Update commit, it's not longer only removed from inside sheet

**Comment by seb-odoo** [comment]
We're going to have another problem. The sheet itself has an overflow hidden too, so it doesn't fix our issue of space for follower list...

**Comment by seb-odoo** [comment]
PS: commit should not be "remove overflow" but "allow overflow"

**Comment by phenix-factory** [comment]
@seb-odoo Done !

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

overflow from sheet in community is fixed in 13.5

**Pull request closed without merging**